commercetools-java-client-apache-async/pom.xml

```diff
@@ -40,6 +40,37 @@
         <plugins>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
```

Comment:
yeah i vebeen able to pull this conf to parent pom but don't know ho to prevent from sharing the test-jar to maven central.